linux/packages/fedora34/audacity.spec

```diff
@@ -84,10 +84,6 @@ supports PulseAudio, OSS and ALSA under Linux.
 %cmake \
     -D CMAKE_BUILD_TYPE=Release \
     -D audacity_conan_enabled=Off \
-    -D audacity_conan_allow_prebuilt_binaries=no \
-    -D audacity_conan_force_build_dependencies=yes \
-    -D audacity_lib_preference=system \
```

Comment:
Why did you remove the Fedora preference for system libraries in this CI spec? Keeping this preference is very useful for matching the downstream builds we do on the Fedora infrastructure and repos because we prefer system libraries when possible (and only do local if truly needed).

Reply:
`audacity_conan_enabled=Off` implies that system libraries are used for all the Conan managed dependencies. However, you are correct, that this changes the behavior for LV2, so I should put this line back